TTS/utils/manage.py

```diff
@@ -346,7 +346,10 @@ def create_dir_and_download_model(self, model_name, model_item, output_path):
 
     def check_if_configs_are_equal(self, model_name, model_item, output_path):
         with fsspec.open(self._find_files(output_path)[1], "r", encoding="utf-8") as f:
-            config_local = json.load(f)
+            try:
+                config_local = json.load(f)
+            except:
```

Comment:
A bare `except:` is not a good idea. For instance, a `KeyboardInterrupt` will be caught by it.

Instead, catch the particular exception that `json.load()` would raise for an empty file (`json.JSONDecodeError`).

Reply:
Actually IMO for this case a bare except is exactly what is needed, if you consider what problem this commit is solving.

The problem here is that it completely avoids downloading anything if something went wrong. The worst case scenario here is it re-downloads the files. It doesn't matter what goes wrong during the json.load, if something unexpected happens, it should just redownload the files.

Reply:
IT should avoid downloading anything if something is wrong. Even if you download the remaining files, the model most prob would not work. We should debug why the files empty and fix that. Any ideas ?

Reply:
A bare `except:` is never a good idea, though. `except Exception:` maybe, if you really need to catch everything, but `except:`, never.

Reply:
+1 for that